openassessment/xblock/static/dist/openassessment-lms.js

```diff
@@ -1515,7 +1515,7 @@ eval("__webpack_require__.r(__webpack_exports__);\n/* harmony import */ var reac
ZCB0aGUgZmlyc3QgZWxlbWVudCB0aGF0IG1hdGNoZXMgdGhhdCBzZWxlY3RvciwgYW5kIHRoZW4gY3JlYXRlcyBhIHJlYWN0IHBvcnRhbCB0byBhIGRpdiB1bmRlcm5lYXRoIHRoZSBwYXJlbnQgZWxlbWVudC5cbiAqL1xuICBwYXJlbnRTZWxlY3RvcjogUHJvcFR5cGVzLnN0cmluZyxcbiAgLyoqIGEgc3RyaW5nIG9yIGFuIGVsZW1lbnQgdGhhdCBpcyByZW5kZXJlZCBpbnNpZGUgb2YgdGhlIG1vZGFsIHRpdGxlLCBhYm92ZSB0aGUgbW9kYWwgYm9keS4gKi9cbiAgdGl0bGU6IFByb3BUeXBlcy5vbmVPZlR5cGUoW1Byb3BUeXBlcy5zdHJpbmcsIFByb3BUeXBlcy5lbGVtZW50XSkuaXNSZXF1aXJlZCxcbiAgLyoqIGEgc3RyaW5nIG9yIGFuIGVsZW1lbnQgdGhhdCBpcyByZW5kZXJlZCBpbnNpZGUgb2YgdGhlIG1vZGFsIGJvZHksIGJldHdlZW4gdGhlIHRpdGxlIGFuZCB0aGUgZm9vdGVyLiAqL1xuICBib2R5OiBQcm9wVHlwZXMub25lT2ZUeXBlKFtQcm9wVHlwZXMuc3RyaW5nLCBQcm9wVHlwZXMuZWxlbWVudF0pLmlzUmVxdWlyZWQsXG4gIC8qKiBhbiBhcnJheSBvZiBlaXRoZXIgZWxlbWVudHMgb3Igc2hhcGVzIHRoYXQgdGFrZSB0aGUgZm9ybSBvZiB0aGUgYnV0dG9uUHJvcFR5cGVzLiBTZWUgdGhlIFtidXR0b25Qcm9wVHlwZXNdKGh0dHBzOi8vZ2l0aHViLmNvbS9lZHgvcGFyYWdvbi9ibG9iL21hc3Rlci9zcmMvQnV0dG9uL2luZGV4LmpzeCNMNDApIGZvciBhIGxpc3Qgb2YgYWNjZXB0YWJsZSBwcm9wcyB0byBwYXNzIGFzIHBhcnQgb2YgYSBidXR0b24uICovXG4gIGJ1dHRvbnM6IFByb3BUeXBlcy5hcnJheU9mKFByb3BUeXBlcy5vbmVPZlR5cGUoW1xuICAgIFByb3BUeXBlcy5lbGVtZW50LFxuICAgIFByb3BUeXBlcy5vYmplY3QsIC8vIFRPRE86IE9ubHkgYWNjZXB0IG5vZGVzIGluIHRoZSBmdXR1cmVcbiAgXSkpLFxuICAvKiogc3BlY2lmaWVzIHRoZSBkaXNwbGF5IHRleHQgb2YgdGhlIGRlZmF1bHQgQ2xvc2UgYnV0dG9uLiBJdCBkZWZhdWx0cyB0byBcIkNsb3NlXCIuICovXG4gIGNsb3NlVGV4dDogUHJvcFR5cGVzLm9uZU9mVHlwZShbUHJvcFR5cGVzLnN0cmluZywgUHJvcFR5cGVzLmVsZW1lbnRdKSxcbiAgLyoqIGEgZnVuY3Rpb24gdGhhdCBpcyBjYWxsZWQgb24gY2xvc2UuIEl0IGNhbiBiZSB1c2VkIHRvIHBlcmZvcm0gYWN0aW9ucyB1cG9uIGNsb3Npbmcgb2YgdGhlIG1vZGFsLCBzdWNoIGFzIHJlc3RvcmluZyBmb2N1cyB0byB0aGUgcHJldmlvdXMgbG9naWNhbCBmb2N1c2FibGUgZWxlbWVudC4gKi9cbiAgb25DbG9zZTogUHJvcFR5cGVzLmZ1bmMuaXNSZXF1aXJlZCxcbiAgdmFyaWFudDogUHJvcFR5cGVzLnNoYXBlKHtcbiAgICBzdGF0dXM6IFByb3BUeXBlcy5zdHJpbmcsXG4gIH0pLFxuICAvKiogc3BlY2lmaWVzIHdoZXRoZXIgdGhlIGRlZmF1bHQgY2xvc2UgYnV0dG9uIGlzIHJlbmRlcmVkIGluIHRoZSBmb290ZXIuIEl0IGRlZmF1bHRzIHRvIHRydWUuICovXG4gIHJlbmRlckRlZmF1bHRDbG9zZUJ1dHRvbjogUHJvcFR5cGVzLmJvb2wsXG4gIC8qKiBzcGVjaWZpZXMgd2hldGhlciBhIGNsb3NlIGJ1dHRvbiBpcyByZW5kZXJlZCBpbiB0aGUgbW9kYWwgaGVhZGVyLiBJdCBkZWZhdWx0cyB0byB0cnVlLiAqL1xuICByZW5kZXJIZWFkZXJDbG9zZUJ1dHRvbjogUHJvcFR5cGVzLmJvb2wsXG4gIC8qKlxuICAgKiBTcGVjaWZpZXMgb3B0aW9uYWwgY2xhc3NlcyB0byBhZGQgdG8gdGhlIGVsZW1lbnQgd2l0aCB0aGUgJy5tb2RhbC1kaWFsb2cnIGNsYXNzLiAgU2VlIEJvb3RzdHJhcCBkb2N1bWVudGF0aW9uIGZvciBwb3NzaWJsZSBjbGFzc2VzLiAgU29tZSBvcHRpb25zOiBtb2RhbC1sZywgbW9kYWwtc20sIG1vZGFsLWRpYWxvZy1jZW50ZXJlZFxuICAgKi9cbiAgZGlhbG9nQ2xhc3NOYW1lOiBQcm9wVHlwZXMuc3RyaW5nLFxufTtcblxuTW9kYWwuZGVmYXVsdFByb3BzID0ge1xuICBvcGVuOiBmYWxzZSxcbiAgcGFyZW50U2VsZWN0b3I6ICdib2R5JyxcbiAgYnV0dG9uczogW10sXG4gIGNsb3NlVGV4dDogJ0Nsb3NlJyxcbiAgdmFyaWFudDoge30sXG4gIHJlbmRlckRlZmF1bHRDbG9zZUJ1dHRvbjogdHJ1ZSxcbiAgcmVuZGVySGVhZGVyQ2xvc2VCdXR0b246IHRydWUsXG4gIGRpYWxvZ0NsYXNzTmFtZTogdW5kZWZpbmVkLFxufTtcblxuZXhwb3J0IGRlZmF1bHQgTW9kYWw7XG4iXSwic291cmNlUm9vdCI6IiJ9\n//# sourceURL=webpack-internal:///./node_modules/@edx/paragon/dist/Modal/index.js\n");
```

Comment:
I split this into lines and diff-ed it and the only thing changed on this (huge) line is:

`[buttonPropTypes](https://github.com/edx/paragon/blob/master/src/Button/index.jsx#L40)`
to
`[buttonPropTypes](https://github.com/openedx/paragon/blob/master/src/Button/index.jsx#L40)`

Reply:
🙏🏻 thank you!